packages/features/settings/appDir/SettingsLayoutAppDirClient.tsx

```diff
@@ -723,7 +724,8 @@ export default function SettingsLayoutAppDirClient({
         <MobileSettingsContainer onSideContainerOpen={() => setSideContainerOpen(!sideContainerOpen)} />
       }>
       <div className="flex flex-1 [&>*]:flex-1">
-        <div className="mx-auto max-w-full justify-center lg:max-w-3xl">
+        <div
+          className={classNames("mx-auto max-w-full justify-center lg:max-w-3xl", rest.containerClassName)}>
```

Comment:
Effort to be up-to-date with `SettingsLayout.tsx` (used for pages router layout)